packages/flutter_tools/lib/src/cmake.dart

```diff
@@ -40,12 +40,12 @@ file(TO_CMAKE_PATH "$escapedProjectDir" PROJECT_DIR)
 
 # Environment variables to pass to tool_backend.sh
 list(APPEND FLUTTER_TOOL_ENVIRONMENT
-  "FLUTTER_ROOT=\\"$escapedFlutterRoot\\""
```

Comment:
Will changing these paths break existing projects that don't already have VERBATIM in their templates?

Also, you need to update the test expectation for this

Reply:
> Will changing these paths break existing projects that don't already have VERBATIM in their templates?

I thought about the same first, but I saw that both `buildLinux()` and `buildWindows()` are unconditionally re-writing `generated_config.cmake`.

> Also, you need to update the test expectation for this

Thanks, done

Reply:
> Will changing these paths break existing projects that don't already have VERBATIM in their templates?

Ugh, good point. We'll need to either write a migrator, or make this logic conditional on the presence of `VERBATIM`.

> but I saw that both `buildLinux()` and `buildWindows()` are unconditionally re-writing `generated_config.cmake`.

Right, but the problem is that `CMakeLists.txt` is only created at `flutter create`-time, and that's where the VERBATIM needs to be added.

Reply:
Right. What kind of migrator did you have in mind? I wonder if it's really worth the trouble. Perhaps going back to the earlier solution (unescaping `'\ '`) would be good enough? It fixes the most common case in a simple and backward-compatible way.

Reply:
The tool already has a migrator API used for iOS but not yet for linux.

See https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/ios/migrations/ios_migrator.dart and https://github.com/flutter/flutter/blob/f865ac7e259d320d7e565f887f894f293a463782/packages/flutter_tools/lib/src/ios/migrations/xcode_build_system_migration.dart#L13